### PR TITLE
Beta fixes 3

### PR DESCRIPTION
**Pull request by jonzee**

### CODE REVIEW

File: src/ts/Contact/Reaction.tsx [hunk 1]
```diff
@@ -22,16 +22,22 @@ export default function Reaction(props){
...
-      autoScrollBodyContent={true}
-      contentStyle={{ minWidth: 320 }}
-   >
-      <div className="reaction">
-        {data.body}
+      <Dialog
+        modal={false}
+        actions={actions}
+        open={open}
+        title={data.title || <div className="dialog-raido"><Raido /></div>}
```
Comment (michalmikolajczyk) [inline src/ts/Contact/Reaction.tsx]: parentheses

File: src/ts/Contact/Reaction.tsx [hunk 2]
```diff
@@ -22,16 +22,22 @@ export default function Reaction(props){
...
+        repositionOnUpdate={false}
+        autoDetectWindowHeight={false}
+        autoScrollBodyContent={false}
+        className="dialog-root"
+        contentClassName="dialog-content"
+        bodyClassName="dialog-body"
+     >
```
Comment (michalmikolajczyk) [inline src/ts/Contact/Reaction.tsx]: space

File: src/ts/helpers/polyfills.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
+// Promise polyfill
+import Promise from 'promise-polyfill';
+if (!window['Promise']) window['Promise'] = Promise;
```
Comment (michalmikolajczyk) [inline src/ts/helpers/polyfills.ts]: what if there is no `window` global available?

Comment (jonzee) [inline src/ts/helpers/polyfills.ts]: I'm using polyfills only in `app.ts` (frontend) and it's not used on server side, but I added additional check anyway.

**Pull request merged**